tests/integration/backward_compatible/proxy/topic_test.py

```diff
@@ -1,5 +1,6 @@
+from hazelcast.errors import NullPointerError
```

Comment:
Not used anymore

Reply:
Removed at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/b91a6e5018e63f4a14938e3680d9762d4586863e.